cart-service/src/tests/service-endpoint.tests.ts

```diff
@@ -244,6 +244,125 @@ test('should return action to set coupon codes on cart when Dovetech service ret
   });
 });
 
+test('should return set commitId action when type is Order', async () => {
```

Comment:
The action returned is still `setCustomType`, so maybe change the name to be a bit clearer. E.g.
```suggestion
test('should return commitId when type is Order', async () => {
```

Reply:
renamed